fancy_dataclass/toml.py

```diff
@@ -155,7 +155,8 @@ def _is_nested(val: Any) -> bool:
                 # TODO: handle None values (comment with empty RHS)
                 settings = self._field_settings(fld).adapt_to(DictDataclassFieldSettings)
                 if settings.doc is not None:
-                    doc.add(tk.comment(str(settings.doc)))
+                    for line in settings.doc.splitlines():
+                        doc.add(tk.comment(line) if line.strip() else tk.nl())
```

Comment:
Doesn't matter too much, but I'm inclined to make this:

```suggestion
                        doc.add(tk.comment(line))
```

so that within-comment newlines still present as being part of the same comment.

Reply:
Of thats what you prefer, it's fine for me